src/SphereXProtectedProxy.sol

```diff
@@ -0,0 +1,68 @@
+// SPDX-License-Identifier: UNLICENSED
+// (c) SphereX 2023 Terms&Conditions
+
+pragma solidity ^0.8.0;
+
+import {Proxy} from "openzeppelin/Proxy/Proxy.sol";
+import {Address} from "openzeppelin/Utils/Address.sol";
+
+import {SphereXProtectedBase} from "./SphereXProtectedBase.sol";
+
+abstract contract SphereXProtectedProxy is SphereXProtectedBase, Proxy {
+    bytes32 private constant PROTECTED_SIG_BASE_POSITION =
```

Comment:
I really hate the SIG terminolegy. Its a protected Function or protected EP (entry points). The sig is not the one being protected